### PR TITLE
lazy load convert_pv jit function

### PR DESCRIPTION
**Pull request by grimoire**
`_NV_CAP = torch.cuda.get_device_capability()` would raise runtime error when torch and cuda driver version mismatch.
And it would bypass `env_check` if we put it in global field.

### CODE REVIEW

File: lmdeploy/pytorch/kernels/pagedattention.py [hunk 1]
```diff
@@ -229,21 +229,27 @@ def _reduce_split_kernel(
...
 
-    @triton.jit
-    def _convert_pv(p, v):
-        """convert pv."""
-        v = v.to(p.dtype)
-        return p, v
+    return convert_pv
+
+
+_convert_pv = None
```
Comment (lvhan028) [inline lmdeploy/pytorch/kernels/pagedattention.py:252]: what `pv` refers to?

Comment (grimoire) [reply lmdeploy/pytorch/kernels/pagedattention.py:252]: `pv = softmax(q, k)@v`

**Review by RunningLeon — APPROVED** [review]
LGTM

**Review by lvhan028 — APPROVED** [review]
